src/include/misc.h

```diff
@@ -164,10 +164,6 @@
  * They come in 3 flavors: F_XXX (handles a field named "flags" in the structure referenced by its
  * argument), LF_XXX (handles a local variable named "flags"), and FLD_XXX (handles any variable,
  * anywhere).
- *
- * Flags are unsigned 32-bit values -- we cast to keep the compiler quiet (the hex constant might be
```

Comment:
The bulk of this paragraph remains relevant, it's just that flags fields can be different sizes that should be changed.